modules/virt-about-live-migration.adoc

```diff
@@ -13,4 +13,9 @@ You can use live migration if the following conditions are met:
 
 * Shared storage with `ReadWriteMany` (RWX) access mode.
 * Sufficient RAM and network bandwidth.
-* If the virtual machine uses a host model CPU, the nodes must support the virtual machine's host model CPU.
\ No newline at end of file
+* If the virtual machine uses a host model CPU, the nodes must support the virtual machine's host model CPU.
+
+[NOTE]
+====
```

Comment:
Location is better but I am not sure this requires a 'note' markup. I would like to see a preview build so that I can check the context,

Reply:
I tend to agree. The fact that encryption is used is not generally "noteworthy", however, it should be mentioned.
IOW if we put it into a note, then it could lead to the impresison that encryptioN/security is special, but it isn't, it should be the default.

Reply:
Also, adding to Fabian's comment -- I have started to look for false "red flags", that is, notes or headers that might cause the user to assume the opposite is true. Hypothetical example: "Functions that require root" header, followed by a paragraph saying that functions that require root are not supported. We don't want the user to think "OMG, my pods are vulnerable to escalation-of-privilege attacks" when we want to assure them that their system is secure.